homeassistant/components/knx/climate.py

```diff
@@ -278,8 +283,6 @@ def hvac_mode(self) -> Optional[str]:
         """Return current operation ie. heat, cool, idle."""
         if self.device.supports_on_off and not self.device.is_on:
             return HVAC_MODE_OFF
-        if self.device.supports_on_off and self.device.is_on:
-            return HVAC_MODE_HEAT
```

Comment:
devices that support only on_off would return HVAC_MODE_OFF as only operation mode, leaving them unusable.